test/e2e/common/k8s.py

```diff
@@ -53,6 +55,78 @@ def to_short_resource_string(self):
     def to_long_resource_string(self):
         return f"{self.plural}.{self.version}.{self.group}/{self._printable_namespace}:{self.name}"
 
+def load_resource(service_name: str,
+                  spec_file: str,
+                  replacements: object):
+    """
+    Load a yaml spec to memory from root_test_path/{service}/resources and replace the values in replacement dict
+    
+    :param service_name: name of service
+    :param spec_file: Name of the spec file under resources directory of the service
+    :param replacements: A dictionary of values to be replaced
+
+    :return: spec as json object
```

Comment:
Hmm this is in a different format to the docstrings I have already started. I have been using the Google style as the template: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
ahh, l just copied from one of the methods and followed it

let's focus on having documentation